betterconf/caster.py

```diff
@@ -7,16 +7,18 @@ def cast(self, val: str):
         raise NotImplementedError()
 
 
-class BoolCaster(AbstractCaster):
-
-    ABLE_TO_CAST = {"true": True, "false": False}
-
-    def cast(self, val: str) -> typing.Union[str, bool]:
-        as_bool = self.ABLE_TO_CAST.get(val.lower())
-        if not as_bool:
+class ConstantCaster(AbstractCaster, typing.Generic[VT]):
```

Comment:
It looks good but i think we should add a empty dictionary to `ConstantCaster`.

Reply:
I've added it.

Reply:
> I've added it.

Awesome!